test_select_algorithm.py TestSelectAlgorithm.test_convolution1 fails on H100 with:

> Mismatched elements: 19584 / 23120 (84.7%)
> Greatest absolute difference: 132.32015991210938 at index (0, 22, 4, 13) (up to 0.0001 allowed)
> Greatest relative difference: inf at index (0, 0, 1, 0) (up to 0.0001 allowed)

See https://github.com/pytorch/pytorch/issues/143412

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_select_algorithm.py b/test/inductor/test_select_algorithm.py
index b30cdc2d946..25d3c068133 100644
--- a/test/inductor/test_select_algorithm.py
+++ b/test/inductor/test_select_algorithm.py
@@ -27,6 +27,7 @@ from torch.testing._internal.common_utils import IS_LINUX, skipIfRocm, skipIfXpu
 from torch.testing._internal.inductor_utils import (
     GPU_TYPE,
     HAS_GPU,
+    IS_H100,
     requires_gpu,
     requires_triton,
 )
@@ -295,6 +296,7 @@ class TestSelectAlgorithm(TestCase):
         foo(torch.randn(64, 64, device=GPU_TYPE))
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
 
+    @unittest.skipIf(IS_H100, "Fails on H100, see #143412")
     @expectedFailureDynamicWrapper
     @patches
     def test_convolution1(self):
